Create placeholder pages for LLM TOC.

xcheng16 opened the pull request:
Differential Revision: D54878408

svekars commented on docs/source/index.rst:
```diff
@@ -79,6 +79,17 @@ Topics in this section will help you get started with ExecuTorch.
+   llm-introduction
+   llm-on-mobile
+   llm-on-desktop
+   llm-on-advanced-flow
```
```suggestion
   llm-advanced-flow
```

svekars commented on docs/source/index.rst:
```diff
@@ -79,6 +79,17 @@ Topics in this section will help you get started with ExecuTorch.
 
+.. toctree::
+   :glob:
+   :maxdepth: 1
```
Needs to be maxdepth
```suggestion
   :maxdepth: 2
```

Commit:
Create placeholder pages for LLM TOC. (#2419)

Summary:


Create the placeholder doc pages following structures in [Purpleberry -  Table Of Contents](https://fburl.com/gdoc/ymk7uxm4). 

https://docs-preview.pytorch.org/pytorch/executorch/2419/index.html

Reviewed By: Jack-Khuu

Differential Revision: D54878408
